CI/travis.linux.after_success.sh

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
 
-set -e
+# set -e
+set -x
+
+[ -n "$TRAVIS_REPO_SLUG" ] && BUILD_DIR="${TRAVIS_BUILD_DIR}" || BUILD_DIR="${BUILD_FOLDER}"
```

Comment:
Is this and the next line some copy/paste left overs? They do not seem to achieve anything...

Reply:
It's actually a bash ternary statement! It sets BUILD_DIR to the right variable.

Reply:
:exploding_head: I hadn't thought of that - but now you mention that I can now see what you mean!